Resources/Prototypes/GameRules/roundstart.yml

```
@@ -324,7 +339,15 @@
   components:
   - type: RampingStationEventScheduler
     scheduledGameRules: !type:NestedSelector
-      tableId: AllGameRulesTable
+      tableId: BasicGameRulesTable
+
+- type: entity
+  id: BasicStationEventSchedulerNoAntag
+  parent: BaseGameRule
+  components:
+  - type: BasicStationEventScheduler
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)